crates/biome_formatter/src/builders.rs

```diff
@@ -1362,9 +1362,10 @@ impl<Context> Group<'_, Context> {
 
 impl<Context> Format<Context> for Group<'_, Context> {
     fn fmt(&self, f: &mut Formatter<Context>) -> FormatResult<()> {
-        let mode = match self.should_expand {
-            true => GroupMode::Expand,
-            false => GroupMode::Flat,
+        let mode = if self.should_expand {
```

Comment:
This is a style rule, really pedantic and can create friction to new contributors. What do you think?

Reply:
This improves readability?

I am also thinking about enabling [redundant_else](https://rust-lang.github.io/rust-clippy/master/#/redundant_else) and [manual_let_else](https://rust-lang.github.io/rust-clippy/master/#/manual_let_else) in an follow-up PR.

Reply:
It's debatable. I like the match very much. I think we should avoid stylistic rules as much as we can, mainly because there's no specific reason to impose a style on new contributors without a good reason (performance, idiomatic, etc.).